release_docs/USING_CMake_Examples.txt

```diff
@@ -12,36 +12,90 @@ Notes: This short instruction is written for users who want to quickly
        More information about using CMake can be found at the Kitware
        site, www.cmake.org.
 
-       CMake uses the command line and these instructions use the script
-       method of the ctest command.
+                     Table of Contents
+
+Section I:    Preconditions
+Section II:   Building HDF5 Examples with CMake
+              1. Using presets
+              2. Using CTestScript.cmake
+              3. Using Command Line CMake
+Section III:  Defaults in the CMakePresets.json file
+Section IV:   Defaults in the HDF5_Examples_options.cmake file
 
 
 ========================================================================
 I. Preconditions
 ========================================================================
 
    1. We suggest you obtain the latest CMake for your platform from the Kitware
-      web site. The HDF5 2.0.x product requires a minimum CMake version
+      web site. The HDF5 2.y.x product requires a minimum CMake version
       of 3.18. If you are using VS2022, the minimum CMake version is 3.21.
+      CMakePresets.json, requires CMake 3.25 or higher.
```

Comment:
Rephrase this so as not to hardcode versions

Reply:
Cannot rephrase - versions are hard minimums - upgrading is only way to remove them.

Reply:
Unless you mean the hdf5 version - then yes.